src/Api/Auth/Models/Request/Accounts/MasterPasswordUnlockDataModel.cs

```diff
@@ -0,0 +1,63 @@
+﻿#nullable enable
+
+using System.ComponentModel.DataAnnotations;
+using Bit.Core.Enums;
+using Bit.Core.KeyManagement.Models.Data;
+
+namespace Bit.Api.Auth.Models.Request.Accounts;
+
+public class MasterPasswordUnlockDataModel : IValidatableObject
+{
+    public required KdfType KdfType { get; set; }
+    public required int KdfIterations { get; set; }
+    public int? KdfMemory { get; set; }
+    public int? KdfParallelism { get; set; }
+
+    public required string Email { get; set; }
+    [StringLength(300)]
+    public required string MasterKeyAuthenticationHash { get; set; }
+    public required string MasterKeyEncryptedUserKey { get; set; }
```

Comment:
```suggestion
    [EncryptedString] public required string MasterKeyEncryptedUserKey { get; set; }
```
Should some of these have the `[EncryptedString]` validator?